tests/functional/fixtures/authentication.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+__all__ = (
+    "user",
+    "with_logged_in_user",
+    "with_logged_in_staff_member",
+    "with_logged_in_admin",
+)
```

Comment:
An all here prevents us from re-exporting `pytest` which looks odd in other places that import things.